cengage/react-magma
# fix(CharacterCounter): Replace `maxLength` prop with `maxCount` and add `hasCharacterCounter` prop

Issue: # [1037](https://github.com/cengage/react-magma/issues/1037)

## What I did
- Added two new props of 'maxCount' replacing 'maxLength' for enabling the Character Counter and 'hasCharacterCounter'. If 'hasCharacterCounter' is set to false, it hides the Character Counter.

## Checklist 
- [x] NA

## How to test

Test both the regular **Input** and **Textarea** components:
- Confirm the Character Counter appears when `maxCount` is set.
- Confirm `maxLength` still works as a Character Counter.
- Confirm `maxLength` works as a native maxlength when `hasCharacterCounter` is set to false.
